python/test/unit/language/test_conversions.py

```diff
@@ -366,3 +366,77 @@ def test_typeconvert_downcast(src_dtype, dst_dtype, rounding, max_repr, device):
 
     for i in range(256):
         downcast_test(getattr(tl, src_dtype), getattr(tl, dst_dtype), rounding, *stuff, max_repr, i, device=device)
+
+@pytest.mark.parametrize("src_dtype, dst_dtype, rounding", [
+    ('float32', 'float8e4nv', 'rtne'),
+    ('float16', 'float8e4nv', 'rtne'),
+    ('bfloat16', 'float8e4nv', 'rtne'),
+    ('float32', 'float8e5', 'rtne'),
+    ('float16', 'float8e5', 'rtne'),
+    ('bfloat16', 'float8e5', 'rtne'),
+])
```

Comment:
You can write this as separate `pytest.mark.parametrize` for each parameter instead of manually flesh all combinations out?